crates/glaredb/src/rpc_proxy.rs

```diff
@@ -1,12 +1,14 @@
-use anyhow::{anyhow, Result};
+use anyhow::Result;
 use protogen::gen::rpcsrv::service::execution_service_server::ExecutionServiceServer;
 use proxyutil::cloudauth::CloudAuthenticator;
 use rpcsrv::proxy::RpcProxyHandler;
 use std::net::SocketAddr;
-use std::path::PathBuf;
 use tonic::transport::{Identity, Server, ServerTlsConfig};
 use tracing::{debug_span, info};
 
+const CERT_PATH: &str = "/etc/certs/tls.crt";
+const CERT_KEY_PATH: &str = "/etc/certs/tls.key";
```

Comment:
Do we know if these are always available? And can we leave a comment letting us know where they're coming from (cert manager?)?